### ISSUE TITLE
best pre-trained weights for 1-class custom data training using yolov3-tiny

### ISSUE THREAD

**Issue by samohadid** [AlexeyAB/darknet#7520]
Hi @AlexeyAB 
I'm confused about which pre-trained weights file should I use when training a 1-class customized dataset, yolov3-tiny.conv.15 or darknet.53.conv.74?
Note that I am using yolov3-tiny.cfg architecture
I hope you can help me 
Thanks

**Comment by stephanecharette** [comment]
See this:  https://www.ccoderun.ca/programming/darknet_faq/#training_command

**Comment by lsd1994** [comment]
yolov3-tiny.conv.15

**Comment by samohadid** [comment]
@stephanecharette 
Hi , in @AlexeyAB darknet Repo we cannot start training without giving a pre-trained model to start from , thus I cannot start training from scratch

**Comment by stephanecharette** [comment]
You are wrong.  I've never once given it pre-trained model to start from.  See the link I posted 2 comments above.

**Comment by samohadid** [comment]
@stephanecharette 
Thanks I will try it again and show you the result.
Is it better always to start from scratch or just if my target object is far from those in COCO dataset ?

**Comment by samohadid** [comment]
@stephanecharette 
Sorry Man it actually worked !
I will wait until it finishes training and tell you which model (with/without pre-trained weight) gives me better result.

**Comment by stephanecharette** [comment]
If it is trained for enough iterations, they should both give more-or-less the same results.
